spec/tunes/app_version_spec.rb

```diff
@@ -28,26 +28,27 @@
       expect(version.can_beta_test).to eq(true)
       expect(version.version).to eq('0.9.13')
       expect(version.supports_apple_watch).to eq(false)
-      expect(version.app_icon_url).to eq('https://is3-ssl.mzstatic.com/image/thumb/Purple3/v4/02/88/4d/02884d3d-92ea-5e6a-2a7b-b19da39f73a6/pr_source.png/1024x1024ss-80.png')
-      expect(version.app_icon_original_name).to eq('AppIconFull.png')
-      expect(version.watch_app_icon_url).to eq('https://muycustomurl.com')
-      expect(version.watch_app_icon_original_name).to eq('OriginalName.png')
+      expect(version.large_app_icon.url).to eq('https://is3-ssl.mzstatic.com/image/thumb/Purple3/v4/02/88/4d/02884d3d-92ea-5e6a-2a7b-b19da39f73a6/pr_source.png/1024x1024ss-80.png')
+      expect(version.large_app_icon.original_file_name).to eq('AppIconFull.png')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.